index.html

```diff
@@ -87,6 +87,27 @@ <h2>Em destaque</h2>
             <div id="img_destaques">
                 
             </div>
+        </section>
+        <section id="secao3">
+            <div class="box">
+                <img src="teste/exemplo.jpg" alt="teste">
+                
+             </div>
+             <div class="box">
+                 <img src="teste/exemplo.jpg" alt="teste">
+                 
+             </div>
+             <div class="box">
+                 <img src="teste/exemplo.jpg" alt="teste">
+                 
+                 
+             </div>
+             <div class="box">
+                 <img src="teste/exemplo.jpg" alt="teste">
+                 

```

Comment:
you must close div and img  here
                 <img src="teste/exemplo.jpg" alt="teste" / > </div>